rotary_utils/assembly.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python
+# assembly.py
+# Module containing functions related to processing Flye-based sequence assembly
+# Copyright Jackson M. Tsuji and Lee H. Bergstrand 2024
+
+import logging
+import pandas as pd
+
+logger = logging.getLogger(__name__)
+
+
+class AssemblyInfo:
```

Comment:
@LeeBergstrand Moved a few functions to a new module called `assembly.py`. The `repair.py` file is still pretty long, but I had a hard time figuring out sensible ways to group the remaining functions into separate modules. We could always revisit this in the future if we want to further shorten `repair.py` -- once more modules are created in rotary, it might become more clear what functions could be moved out of `repair.py` to other shared modules. Let me know if you are OK with the length of `repair.py` for now or if it would be better to move more of the functions out to separate modules before merging this PR -- thanks.